services/base-service/models/expert/api.js

```diff
@@ -279,47 +168,28 @@ router.route(
   }
 );
 
+function expert_valid_path(options) {
+  const def = {
+    "description": "Get an expert by id",
+    "parameters": [
+      "#components/parameters/expertId"
+    ],
+    "responses": {
+      "Expert": "#/components/responses/Expert",
```

Comment:
I don't think I fully understand what I'm really supposed to do with these "$ref".  You can put two in the same object, they overwrite eg. so what is valid?